### DISCUSSION TITLE
Spatial aggregation

### DISCUSSION THREAD

**Discussion by ddjustina** [aazuspan/wxee#48] [Q&A] [answered]
I'm not very proficient in python, so I would like to know if it's possible to get a time series table by region or by points, given a feature collection. Thanks!

**Comment by aazuspan** [comment, marked as answer]
This is definitely possible with Earth Engine and Python. Check out [this notebook](https://github.com/renelikestacos/Google-Earth-Engine-Python-Examples/blob/master/005_EE_Proba-V_NDVI_Time-Series_Analysis.ipynb) which walks through how you can extract a time series dataframe of NDVI data over a point.

You won't need `wxee` just to extract a time series of data, but might find it useful if you need to modify the time series within Earth Engine, e.g. aggregating daily data to monthly, filling gaps, etc.
